app/views/layouts/application.html.erb

```diff
@@ -1,9 +1,13 @@
+<% mascot = current_user&.mascot || 'ki' %>
```

Comment:
I wanted to consolidate the two places where the logged-out mascot defaults to Ki. Is this the most appropriate place to put this line?

Reply:
Just added find_pornlizard to the User model, I'll switch over to that

Reply:
Ehhhh nevermind, not worth it actually, since current_user is nillable